executable on the same folder as the assets (#6)
Stepland/jujube: docs/Compiling.md

    $ ninja
    ```
    (Replacing the `{...}`s with what you chose)

> 0. Copy `jujube` executable from the build folder into project is root folder.

Reviewer: ```suggestion
0. Copy the resulting `jujube` executable from the build folder to the project root folder.
```
